_posts/2016-12-06-recurse-center-week-10.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Week[9] @ The Recurse Center 🔎"
+comments: true
+categories: tech recurse
+---
+
+### Overall
+
+I'm currently trying to distill through what is essential, and focus on that. Stripping away the excess has left more energy for the important pursuits.
+
+### Emotional 😰
+
+### **Frantic**
+
+Feels have been evolving recently, but at the moment I feel frantic. There's now only **2** weeks left.
+
+I was reflecting on the first half versus the second half. It seems like I got more done in the first half...
+So far I've done a bunch into Haskell but haven't made progress on any **projects**.
+
+I feel disappointed that I haven't built some "Pièce De Résistance". I'll have completed all the Haskell resources, but typing out function definitions isn't *sexy*.
+
+I'm feel frantic to find something I can build in two weeks that will prove that **I am better**. A **DRASTICALLY** better programmer.
+
+This desire has also motivated me to constantly be working.
+
+> It's 9pm... but I don't feel accomplished. Gah! Stay later, work longer, we'll be drastically better soon.
+
+Maybe if I went back 3 months, the improvement would be more obvious. I feel drastically better than a year ago, but it's hard to know how much of that has come from RC.
+
+Anyway... I'm frantic to feel accomplished, and it hasn't hit me yet.
```

Comment:
One time where I recently felt that I'd drastically improved as a programmer was when I revisited blog posts that had previously gone completely over my head and I was able to comprehend them.

Remember also that you'll still be able to build things after your batch is over. In fact, some of my most productive times during my batch were when I wasn't even at RC. It is easier for me to sit down and write code when I'm not distracted by the opportunity to hang out with 70 other lovely people :smile:

In short: I wouldn't worry too much about being productive.